docs/definitions/award_suppliers_summary.csv

```diff
@@ -1,18 +1,18 @@
 Column Name,Data Type,Description
 id,bigint,"Unique id representing a release, compiled_release or record"
-award_index,bigint,
-supplier_index,bigint,
-release_type,text,"Either release, compiled_release or record. compiled_release are releases generated by kingfisher release compilation"
+award_index,bigint,Position of the award in the awards array
+supplier_index,bigint,Position of the supplier in the supplier array
+release_type,text,"Either release, compiled_release or record. compiled_releases are releases generated by kingfisher release compilation"
 collection_id,integer,id from kingfisher collection table
 ocid,text,ocid from the data
-release_id,text,Release id from the data. Relevent for releases and not for compiled_releases or records
+release_id,text,Release id from the data. Relevant for releases and not for compiled_releases or records
 data_id,integer,"id for the ""data"" table in kingfisher that holds the original JSON data."
-supplier,jsonb,
-supplier_parties_id,text,
-supplier_identifier,text,
-unique_identifier_attempt,text,
-supplier_additionalidentifiers_ids,jsonb,
-supplier_additionalidentifiers_count,integer,
-link_to_parties,integer,
-link_with_role,integer,
-party_index,bigint,
+supplier,jsonb,JSONB of supplier object
+supplier_parties_id,text,`id` from supplier object
+supplier_identifier,text,Concatenation of `scheme` and `id` from `identifier` object in the form `<scheme>-<id>`
+unique_identifier_attempt,text,"The `id` from party object if it exists, otherwise the identifier described above if it exists, otherwise the party name"
+supplier_additionalidentifiers_ids,jsonb,JSONB list of the concatenating scheme and id of all additionalidentifier objects
```

Comment:
additionalidentifier -> additional**I**dentifier for consistency with field in standard?